src/box/memtx_space.c

```diff
@@ -883,6 +883,11 @@ memtx_space_check_index_def(struct space *space, struct index_def *index_def)
 			 "covering index");
 		return -1;
 	}
+	if (index_def->opts.layout != NULL) {
+		diag_set(ClientError, ER_UNSUPPORTED, "memtx",
+			 "layout options");
```

Comment:
Nit. Looks like should be `"layout option"`.

Reply:
Changed to `memtx does not support 'layout' option`.